koocook_core/templatetags/recipe_extras.py

```diff
@@ -8,3 +9,29 @@ def tag_level(lv: int) -> str:
     """ Return Bulma color helper by given label_level """
     level = {1: 'is-info', 2: 'is-success', 3: 'is-warning', 4: 'is-danger'}
     return level.get(lv)
+
+
+@register.filter(name='timedel_to_words')
+def duration_in_words(duration: datetime.timedelta) -> str:
+    """Converts a timedelta duration to a word
+
+    Args:
+        duration (datetime.timedelta): A given duration
+
+    Returns:
+        (str) A timedelta duration in word representation
+    """
+    def plural(n):
+        return n, abs(n) != 1 and "s" or ""
```

Comment:
How about using one line if-else instead? I think it should be clearer while still being short.

```python
return n, '' if abs(b) == 1 else 's'
```

It's a lot more readable.

Reply:
Good idea. I'll take action on 'em.